lua/nvim-treesitter-playground/utils.lua

```diff
@@ -65,12 +65,7 @@ function M.get_hl_groups_at_position(bufnr, row, col)
       if hl and ts_utils.is_in_node_range(node, row, col) then
         local c = query._query.captures[capture] -- name of the capture in the query
         if c ~= nil then
-          local general_hl, is_vim_hl = c, false
-          local local_hl = not is_vim_hl and (tree:lang() .. general_hl)
-          table.insert(
-            matches,
-            { capture = c, specific = local_hl, general = general_hl, priority = metadata.priority }
-          )
+          table.insert(matches, { capture = c, priority = metadata.priority })
```

Comment:
previous revision of the PR had `@function -> @function` not sure whether that is clearer. Or should we display `@function.lua`

Reply:
If you want to be explicit, the full chain is (for a fictive example)
```
@function.special.lua -> @function.special -> @function -> TSFunction -> Function
```
(with the `TSFunction` being a compatibility shim we haven't decided what to do with after 0.8). Users can break this chain at any point by `hi link @function.special.lua Foo`, which ideally should be reflected.